.github/actions/set_java_in_container/action.yml

```
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+name: "Sets Java version"
+description: "Sets the Java version within Flink' CI Docker container"
+inputs:
+  jdk-version:
```

nit
```suggestion
  jdk_version:
```
I would suggest to use underscore to have same names as for connector's gha
https://github.com/apache/flink-connector-shared-utils/blob/969edc5814d021d636b5fed1877f7932214be7ea/.github/workflows/ci.yml#L30